### DISCUSSION TITLE
Custom @response does not work ?

### DISCUSSION THREAD

**Discussion by betterapp** [dedoc/scramble#198] [Q&A]
I have something like this:

![image](https://github.com/dedoc/scramble/assets/7449124/22fe43cf-b509-4c34-8a93-db48a160d9a0)

and documentation looks like this:
![image](https://github.com/dedoc/scramble/assets/7449124/9faf6687-0840-485d-9e50-14d7c750cab8)

![image](https://github.com/dedoc/scramble/assets/7449124/85aebabd-2573-4ee7-81b3-b5eb348a735e)

Is this is some old version ?
![image](https://github.com/dedoc/scramble/assets/7449124/ec770a13-29fe-46dd-ac61-d608c7ad609c)

or API documentation is generated on the fly.

How can I regenerate it if it is needed ?

**Comment by romalytvynenko** [comment]
Hey @betterapp 

Docs are generated on the go. You simply update the page with docs to get a fresh version.

This is a bug indeed. Will fix.
